appendix/migrate-to-postgresql.rst

```diff
@@ -0,0 +1,175 @@
+Migrate to PostgreSQL server
+****************************
+
+Support for MySQL/MariaDB will be dropped in Zammad 7.0 upwards. Make sure to
+migrate your existing instance of Zammad to PostgreSQL before that update.
+
+The following guide will provide you with a rough direction through that
+migration process.
+
+.. warning:: **Proof of concept ahead**
+
+   As the technical details may differ from system to system, this guide
+   comes without any warranty. Please proceed at your own risk. In doubt
+   please refer to the documentation of the tools used.
+   
+
+Preparation
+===========
+
+#. Stop Zammad: 
+
+   .. code-block:: sh
+
+      $ systemctl stop zammad
+
+#. Create a backup of your instance.
+
+
+Install PostgreSQL
+------------------
+
+.. include:: /install/includes/postgres-installation.rst
+
+Please also have a look at :doc:`/appendix/configure-database-server`.
+
+.. tabs::
+
+   .. tab:: Package installation
+
+      Nothing to do, continue with the next step. 🎉
+
+   .. tab:: Source code installations
+
+      .. include:: /install/includes/postgres-dependencies.rst
+
+Database Credentials
+--------------------
+
+Get the credentials Zammad is currently using to access your MySQL/MariaDB
+database from ``config/database.yml``. You will need them later.
+
+Now adjust the configuration file to fill in the credentials for your new
+PostgreSQL server. Use ``postgresql`` as ``adapter``.
+
+
+.. include:: /install/includes/postgres-permissions.rst
+
+
+Create empty database
+---------------------
+
+Now you need to create an empty database in PostgreSQL.
+
+.. tabs::
+
+   .. tab:: Source installation
+      
+	  .. code-block:: sh
+	     
+		 $ su - zammad
+		 $ rake db:create
+		 
+   .. tab:: Package installation
+  
+      .. code-block:: sh
+	     
+	  	 $ zammad run rake db:create
+
+
+Install pgloader and migrate
+============================
+
+.. tabs::
+
+   .. tab:: Ubuntu / Debian
+
+      .. code-block:: sh
+
+         $ apt update
+         $ apt install pgloader
+
+   .. tab:: CentOS
+
+      .. code-block:: sh
+
+         $ yum install -y https://download.postgresql.org/pub/repos/yum/common/redhat/rhel-7-x86_64/pgloader-3.6.2-1.rhel7.x86_64.rpm
```

Comment:
Something seemed to went wrong here, can you please adjust to the correct package required?
Or however you install it on CentOS.

Reply:
Works for me on CentOS7 and CentOS8 Stream on fresh Hetzner-Cloud-servers. Can you specify what went wrong?

Reply:
Sorry I can't find what disturbed me yesterday, let's ignore it.

Reply:
Ah wait the wording thing (2 times the same text) still is valid of course.